modules/appendix/pages/openCypher-gsql-from-clause-extension.adoc

```diff
@@ -0,0 +1,324 @@
+= Cypher Pattern Support in GSQL
+
+We have extended GSQL to support Cypher patterns in the FROM clause.
```

Comment:
```suggestion
We have extended GSQL to support openCypher patterns in the FROM clause.
```

Reply:
Changed several places where I said "Cypher" to "openCypher". reflected in the latest commit.